src/commands/Announcement/announcement.ts

```diff
@@ -70,11 +78,11 @@ export default class extends SkyraCommand {
 				const resultMessage = shouldSendAsEmbed
 					? await previous.edit(
 							mentions.length
-								? message.language.get(LanguageKeys.Commands.Announcement.AnnouncementEmbedMentionsWithMentions, {
+								? language.get(LanguageKeys.Commands.Announcement.AnnouncementEmbedMentionsWithMentions, {
 										header,
-										mentions: message.language.list(mentions, message.language.get(LanguageKeys.Globals.And))
+										mentions: language.list(mentions, await message.fetchLocale(LanguageKeys.Globals.And))
```

Comment:
Why are you fetchLocale here when you have language?
```suggestion
										mentions: language.list(mentions, language.get(LanguageKeys.Globals.And))
```